cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/enumeration/CareGapsStatusCode.java

```diff
@@ -31,6 +32,10 @@ public String toDisplayString() {
             return "Not Applicable";
         }
 
+        if (value.equals("prospective-gap")) {
```

Comment:
Always a good idea to flip the direction of these checks in case the left side is null:

"prospective-gap".equals(value)